extensions/elytron-security-ldap/runtime/src/main/java/io/quarkus/elytron/security/ldap/config/AttributeMappingConfig.java

```diff
@@ -0,0 +1,35 @@
+package io.quarkus.elytron.security.ldap.config;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+/**
+ * Configuration information used to populate a {@linkplain org.wildfly.security.auth.realm.ldap.AttributeMapping}
+ */
+@ConfigGroup
+public class AttributeMappingConfig {
+
+    /**
+     * The roleAttributeId from which is mapped (e.g. "cn")
+     */
+    @ConfigItem(name = "from")
```

Comment:
You can remove all the names if the property has the right name.

Reply:
Ok, I'm going to fix this.